File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
startH, startM = input().split()
startH = int(startH)
startM = int(startM)
nT = int(input())

nH = int(nT/60)
nM = int(nT%60)

startM = startM + nM
if(startM>=60):
    startM = startM-60
    startH = startH + 1

startH = startH + nH
if(startH>=24):
    startH = startH-24
print(startH, startM)

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');

// let a = Number(input[0].split(' ')[0]);
// let b = Number(input[0].split(' ')[1]);
let [a, b] = input[0].split(' ').map(Number);
let c = Number(input[1]);

let totalMinute = a * 60 + b + c;
totalMinute %= 1440;
let hour = parseInt(totalMinute/60);
minute = totalMinute % 60;

console.log(hour + " " + minute);